creator/files/models.py

```diff
@@ -369,6 +370,22 @@ def extract_config_path(self):
 
         return config_path
 
+    def set_storage(self):
+        """
+        Set storage location for study bucket if using S3 backend
+        """
+        s3_storage = "django_s3_storage.storage.S3Storage"
+        if settings.DEFAULT_FILE_STORAGE == s3_storage:
+            if self.study is not None:
+                study = self.study
+            elif self.root_file is not None:
+                study = self.root_file.study
+            else:
+                raise Study.DoesNotExist(
+                    f"Version {self} must be part of a study."
```

Comment:
for consistency could import the `version_display` function in this file then do
```
                    f"Version {version_display(self)} must be part of a study."
```